sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/query.py

```diff
@@ -0,0 +1,305 @@
+"""Adjust data - download deliverables, clean and upload to BigQuery."""
+import csv
+import json
+import os
+import tempfile
+from argparse import ArgumentParser
+
+import requests
+from google.cloud import bigquery
+
+API_URI = "api.adjust.com"
+ENDPOINT = "kpis"
+API_VERSION = "v1"
+
+"""This is a list of dictionaries.
```

Comment:
I'm not sure I understand what this comment is trying to tell me. Also, rather than saying that Mozilla has 16 apps in adjust I'd prefer you to mention here which apps this logic might be running for. If there's a link where you could find a list of the apps available and and their descriptions you could include that here.

Reply:
I was being explicit as to what the list contains - the set up is important for how the code runs. I.e. you need to have the list in the form of [{key:app_name, token:app_token}] in order for the code to work.

I've added a bit more clarity